include/sampling/sample_correlation_matrices.hpp

```diff
@@ -0,0 +1,131 @@
+/// Functions to sample correlation matrices w.r.t. a truncated density
+
+#ifndef SAMPLE_CORRELATION_MATRICES_HPP
+#define SAMPLE_CORRELATION_MATRICES_HPP
+
+#include <sampling/sampling.hpp>
+
+/// Direct implementation: call to sampling algorithms for spectrahedra in volesti
+/// Mainly serves for comparison
```

Comment:
if this is not needed in the library or is only for debugging or testing purposes then please do not include it in the library but rather create an example

Reply:
I moved these functions to examples.